src/components/SwipeButtons.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { View, Text, StyleSheet, TouchableOpacity, Button } from 'react-native'
+import { MaterialCommunityIcons } from '@expo/vector-icons';
+import { Feather } from '@expo/vector-icons';
+
+export type SwipeButtonsParamList = {
+    onPress: void
+}
+
+export default function SwipeButtons({onPress}: {onPress: SwipeButtonsParamList}) {
+    return (
+        <View style={styles.container}>
+            <TouchableOpacity style={styles.button} onPress={() => {return onPress}}>
+                <Feather name="x-circle" size={24} color="black" />
+            </TouchableOpacity>
+            <TouchableOpacity style={styles.button} onPress={() => {return onPress}}>
+                <MaterialCommunityIcons name="silverware-fork-knife" size={24} color="black"/>
+            </TouchableOpacity>
+        </View>
+    )
+}
```

Comment:
Maybe you can explain the onPress property sometime? It's currently logging 'hello' from the MenuScreen component. I see... 

Wondering if it makes more sense to pass props to the Swipe Button, so we can make API requests to our backend from the SwipeButtons component OR, to pass a callback like what is done here. Passing props might keep some code out of the MenuScreen that is only associated with the SwipeButtons. But open to suggestions